teemaderegister-be/src/controllers/users.js

```diff
@@ -179,3 +179,9 @@ module.exports.resetPicture = async (req, res) => {
 
   return res.json({ user, message: 'Picture updated successfully' })
 }
+
+module.exports.getAllUsers = async (req, res) => {
+  const users = await User.find({}, {login: 0})
```

Comment:
As the page is for admin only, I would exclude only password and include other columns, email if there is any etc.

As the could be many users would do also pagination – no need to add it now, can be done with separate task later.

Reply:
What do you exactly mean by pagination? There is already pagination already included in the ant.design table or am I misunderstanding something?
 
![pagination](https://user-images.githubusercontent.com/55079581/104141878-90ea4d80-53c1-11eb-98b9-ac667ffd07ff.png)